base/caveexpress/campaigns/10-letters-campaign.lua

```diff
@@ -10,8 +10,3 @@ c:addMaps("letter-02")
 c:addMaps("letter-03")
 c:addMaps("letter-04")
 c:addMaps("letter-05")
-c:addMaps("letter-06")
```

Comment:
is this change intentional?

Reply:
yes, those aren't yet done, nor in repo
Will add them here after I create those maps eventually